ts/Dashboards/Components/DataGridComponent/DataGridComponentDefaults.ts

```diff
@@ -40,11 +40,99 @@ const DataGridComponentDefaults: Globals.DeepPartial<Options> = {
+        }, {
+            name: 'DataGrid options',
+            type: 'nested',
+            nestedOptions: [{
+                name: 'General',
+                options: [
+                    {
+                        name: 'Caption/title',
+                        propertyPath:
+                            ['dataGridOptions', 'settings', 'caption', 'text'],
+                        type: 'input'
+                    }, {
+                        name: 'Columns distribution',
+                        propertyPath:
+                            [
+                                'dataGridOptions',
+                                'settings',
+                                'columns',
+                                'distribution'
+                            ],
+                        type: 'select',
+                        selectOptions: [{
+                            name: 'full'
+                        }, {
+                            name: 'fixed'
+                        }]
+                    }, {
+                        name: 'Editable DataGrid',
+                        propertyPath:
+                            [
+                                'dataGridOptions',
+                                'defaults',
+                                'columns',
+                                'editable'
+                            ],
+                        type: 'toggle'
+                    }, {
+                        name: 'Resizable columns',
+                        propertyPath:
+                            [
+                                'dataGridOptions',
+                                'settings',
+                                'columns',
+                                'resizable'
+                            ],
+                        type: 'toggle'
+                    }, {
+                        name: 'Sortable columns',
+                        propertyPath:
+                            [
+                                'dataGridOptions',
+                                'defaults',
+                                'columns',
+                                'sorting',
+                                'sortable'
+                            ],
+                        type: 'toggle'
+                    }, {
+                        name: 'Cell text truncation',
+                        propertyPath:
+                            [
+                                'dataGridOptions',
+                                'settings',
+                                'rows',
+                                'strictHeights'
+                            ],
+                        type: 'toggle'
+                    }
+                ]
+            }]
+        }, {
+            name: 'DataGrid class name',
+            propertyPath: ['dataGridClassName'],
+            type: 'input'
+        }, {
+            name: 'DataGrid ID',
+            propertyPath: ['dataGridID'],
+            type: 'input'
```

Comment:
These are not updated on unfocus and save, when checking /cypress/component-datagrid.

I my previous review I wrote "we should also add "DataGrid class name" and "DataGrid ID" to the root here". The placement is to be consistent with other components, but I've added a separate task (https://app.asana.com/0/1203234853779898/1208086790358623) for a cleanup here. Let's look at all components together later on.